packages/common/types/account.ts

```diff
@@ -232,31 +204,21 @@ export type CaptureCustomAnswer = {
   value: string;
 };
 
-export type Capture = {
-  firstName?: string;
-  address?: string;
-  address2?: string;
-  city?: string;
-  state?: string;
-  postCode?: string;
-  country?: string;
-  birthDate?: string;
-  companyName?: string;
-  phoneNumber?: string;
-  customAnswers?: CaptureCustomAnswer[];
+export type RegistrationFields = {
+  beforeSignUp: boolean; // true when the registration fields need to be filled in while signing up
```

Comment:
This var name had me questioning. Not sure if I have a better alternative, but some suggestions: `isRequiredInitially`, `isRequiredDuringSignUp`, `isRequiredBeforeSignUp` (although it's more 'during' I'd say), or just `requiredBeforeSignUp` / `requiredDuringSignUp`.

(If you could argue that it being `required` shouldn't be reflected, and it being `before` is more accurate: then leave it as is.)

Reply:
Yes, this name is not final (I thought I already mentioned that somewhere). We could use some explanations/doc blocks in the typings. This should show nicely in the IDE as well.

I did prevent using the term `required` indeed... But `requiredBeforeSignUp` sounds a lot better 😄